client/client.go

```diff
@@ -411,6 +408,7 @@ func (c *Cmd) Dial() error {
 	}
 	if len(c.FSTab) > 0 {
 		c.Env = append(c.Env, "CPU_FSTAB="+c.FSTab)
+		c.Env = append(c.Env, "LC_GLENDA_CPU_FSTAB="+c.FSTab)
```

Comment:
I'm not sure about this, but maybe make it a constant? It's a big string but it used in two different packages and sometimes we need `=` so it maybe a bit awkward...

Reply:
I kinda went back and forth on that, but maybe not just yet, I'm still hoping there is a better way to do this?

Reply:
I think it should be, but it not clear to me either.